crates/subspace-runtime/src/lib.rs

```diff
@@ -93,6 +93,11 @@ sp_runtime::impl_opaque_keys! {
     }
 }
 
+// The domain storage fee multiplier used to charge a higher storage fee to the domain
+// transaction to even out the duplicated/illegal domain transaction storage cost, which
+// can not be eliminated right now.
+const DOMAIN_STORAGE_FEE_MULTIPLIER: Balance = 2;
```

Comment:
I have found an error in my calculation of this multiplier. It was also considering not only the probability of transactions being included 1,2 or more times, but also transactions that get included 0 times. This was incorrect for our use case as those transactions don't pay the fees until they are included. So if we excluded unpicked txs from the picture, the mean duplication rate in 6 slots is closer to 2.4. So I suggest updating this multiplier to 2.5 or even 3 if it's either to work with an integer.

Reply:
Updated to 3 in the latest commit